pkg/operator/certrotation/metrics/monitor.go

```diff
@@ -0,0 +1,187 @@
+package metrics
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	corev1listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/util/cert"
+	"k8s.io/klog"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/openshift/library-go/pkg/crypto"
+)
+
+const (
+	// CertificateManagedTypeLabelName marks config map or secret as object that contains managed certificates.
+	// This groups all objects that store certs and allow easy query to get them all.
+	// Also any object with this label will be collected by prometheus and transformed to metrics.
+	// The value of this label should be set to "true".
+	CertificateManagedTypeLabelName = "auth.openshift.io/certificate-managed-type"
```

Comment:
I think these are `managed-certificates`, so the prefix would be related to managed-certificates for sorting.  As distinct from serving-certificate